postgres_mixin/dashboards/dashboards.libsonnet

```diff
@@ -1,5 +1,5 @@
 {
   grafanaDashboards+:: {
-    'postgres-overview.json': (import 'postgres-overview.json'),
+    'postgresql-overview.json': (import 'postgresql-overview.json'),
```

Comment:
I think this might cause some issues to existing users, best if we keep it as is if possible.